main.rb

```diff
@@ -27,7 +27,7 @@
 
     print "\nhost name? (https://<example.com> /path. Give me <> part plz \n"
     hostname = gets.chomp
-    commands = [{:workflow => "login", :user => user, :input => input}]
+    commands = [{:workflow => "add_home", :user => user, :input => input}]
```

Comment:
This is fine, but should be worked to eventually use this type of input
```
{
  "hostname": "BASE_URL",
  "commands": [
    {
      "workflow": "login",
      "user": {
        "username": "jpinto",
        "password": "LibertyBell"
      },
      "input": {
        "title": "Sales"
      }
    }
  ]
}
```

With the input needed listed in that json
Maybe we can work a way to point that "Input" to another file to be read, assuming its gonna be the in the same folder?

Sorry, I don't think I added that file because I didn't want to put it up on github with the domain